setup.py

```diff
@@ -4,15 +4,13 @@
 import os
 from setuptools import setup
 
-import oled
-
 README = open(os.path.join(os.path.dirname(__file__), "README.rst")).read()
 CONTRIB = open(os.path.join(os.path.dirname(__file__), "CONTRIBUTING.rst")).read()
 CHANGES = open(os.path.join(os.path.dirname(__file__), "CHANGES.rst")).read()
-version = oled.__version__
+version = "2.0.0"
```

Comment:
Namespaced packages - possibly better to have a VERSION.txt (or whatever) file in the root directory and it gets slurped from there